manifests/install.pp

```diff
@@ -117,7 +117,7 @@
   } elsif ($::osfamily == 'RedHat') and ($package_version != 'present') {
     # Workaround PUP-5802/PUP-5025
     if ($::operatingsystem == 'Fedora') {
-      $pkg_os_suffix = 'fedoraf'
+      $pkg_os_suffix = 'fedora'
```

Comment:
I won't be around to work on this the next 2 weeks. Please ask @puppetlabs/agent-team for help.